src/videodownlinkbandwidthpolicy/VideoPriorityBasedPolicy.ts

```diff
@@ -229,6 +241,37 @@ export default class VideoPriorityBasedPolicy implements VideoDownlinkBandwidthP
     }
   }
 
+  setVideoPriorityBasedPolicyConfigs(config: VideoPriorityBasedPolicyConfig): void {
+    this.videoPriorityBasedPolicyConfig = config;
+  }
+
+  private static readonly SHORT_DELAY = VideoPriorityBasedPolicy.MIN_TIME_BETWEEN_SUBSCRIBE_MS;
+  private static readonly LONG_DELAY = 8000;
+
+  // convert network event delay factor to actual delay in ms
+  private getCurrentDelay(event: NetworkEvent, numberOfParticipants: number): number {
+    // convert enum to ms value
+    let minTimeBetweenSubscribe = VideoPriorityBasedPolicy.SHORT_DELAY;
```

Comment:
`updateDelay`, `subscribeDelay`, etc. and change function name to be consistant.